.github/prompts/fix-pr-ci-failures.md

```diff
@@ -24,8 +24,9 @@ Do not stop a given execution until you have worked through all phases below.
 **Phase 1 is for gathering information ONLY. Do NOT analyze failures or look at any code during this phase.**
 **Your only goal in Phase 1 is to collect: job names, job IDs, log files, and failed task names.**
 
-1. Get repository info: `gh repo view --json owner,name`
-2. For unique job type that failed
+1. Delete any existing CI-PLAN.md file: `rm -f CI-PLAN.md`
```

Comment:
a couple of times it has ignored "assume you are starting fresh from the beginning of the process" above and gotten confused by the existing CI-PLAN.md